src/L.Routing.GraphHopper.js

```diff
@@ -109,6 +109,7 @@
 					summary: {
 						totalDistance: path.distance,
 						totalTime: path.time / 1000,
+            totalAscend: path.ascend,
```

Comment:
Could you fix indentation here? The rest of the lib uses tabs (sorry), but this line is indented with spaces.